piqe_ocp_lib/api/resources/ocp_resource_quotas.py

```diff
@@ -0,0 +1,926 @@
+from piqe_ocp_lib.api.resources.ocp_base import OcpBase
+from piqe_ocp_lib.api.ocp_exceptions import ConfigError
+from kubernetes.client.rest import ApiException
+from kubernetes.client import V1ResourceQuota, V1ResourceQuotaSpec, V1ResourceQuotaStatus, V1ObjectMeta, \
+    V1ScopeSelector, V1ScopedResourceSelectorRequirement
+from openshift.dynamic.resource import ResourceList, ResourceInstance
+from typing import List, Union, Dict, Type, Any, Optional
+from enum import Enum, auto
+import logging
+from piqe_ocp_lib import __loggername__
+
+logger = logging.getLogger(__loggername__)
+
+
+class OcpResourceQuota(OcpBase):
+    """
+    OcpResourceQuota Class extends OcpBase and encapsulates all methods
+    related to openshift resource quotas.
+    :param kube_config_file: A kubernetes config file.
+    :return: None
+    """
+    def __init__(self, kube_config_file=None):
+        super(OcpResourceQuota, self).__init__(kube_config_file=kube_config_file)
+        self.api_version = 'v1'
+        self.kind = 'ResourceQuota'
+        self.ocp_resource_quota = self.dyn_client.resources.get(api_version=self.api_version, kind=self.kind)
+
+    def get_resource_quotas(self, namespace: str) -> ResourceList:
+        """
+        Get all ResourceQuotas from specific namespace
+        :param namespace: (str) name of the namespace
+        :return: ResourceQuotas response on Success OR None on Failure
+        """
+        api_response = None
```

Comment:
I was thinking more about this, we might want to consider on whether we want to return None or an empty list on failure if we return a list of objects on success. The rational is to have better clarity and consistency on how we write conditionals around this methods at the script or higher api level. For example, this would be the difference between doing:
if get_resource_quotas(namespace) vs if len(get_resource_quotas(namespace))

Reply:
Agree. I feel like EAFP puts the onus more on the end user which in our case could be the test developer of higher level APIs. Debugging could get trickier in the latter case but for sure there are pros and cons to both. We can add this topic to the agenda in the next meeting.